packages/react-i18n/src/i18n.ts

```diff
@@ -825,4 +833,8 @@ export class I18n {
   private defaultOnError(error: I18nError) {
     throw error;
   }
+
+  private normalizeTimezone(timeZone?: string) {
+    return timeZone ? mapDeprecatedTimezones(timeZone) : undefined;
```

Comment:
We initially had a test added to this - but the tests were actually a false positive, testing date for date, the test for the mapping is actually seen in 

https://github.com/Shopify/quilt/blob/main/packages/dates/src/tests/map-deprecated-timezones.test.ts